src/appConfig.js

```diff
@@ -80,7 +80,8 @@ configuration.queryparams =
         'GageStatsServicesBounds': '/stations/Bounds?xmin={0}&xmax={1}&ymin={2}&ymax={3}&geojson=true',
         'FlowAnywhereEstimates': '/models/FLA/estimate?state={0}',
         'FlowAnywhereGages': '/arcgis/rest/services/IowaStreamEst/FlowAnywhere/MapServer/1/query?geometry={0},{1}&geometryType=esriGeometryPoint&inSR=4326&spatialRel=esriSpatialRelIntersects&outFields=regions_local.Region_Agg,reference_gages.site_id,reference_gages.site_name,reference_gages.da_gis_mi2,reference_gages.da_pub_mi2,reference_gages.lat_dd_nad,reference_gages.long_dd_na&returnGeometry=false&returnIdsOnly=false&returnCountOnly=false&returnZ=false&returnM=false&returnDistinctValues=false&f=pjson',
-        'NSSRegions': '/regions/'
+        'NSSRegions': '/regions/',
+        'NSSRegionScenarios': '/regions/{0},NA/scenarios/'
```

Comment:
> This is looking good! In addition to my small comments, please delete or comment out console.logs just as a good practice.
> 
> One thing we can figure out either now or later is that we want each flow statistics checkbox to be associated with its statisticGroupID value, because that numeric value is what we'd like to send to the BP service. We may want to have one big ng-model for all the checkboxes so that later we can easily grab all the statisticGroupIDs of all the checked flow statistics.

Building on this comment and @harper-wavra, I tried coding this on Friday evening and didn't have any luck. I'll try again based on @harper-wavra code suggestion just so the feature is ready for the next issue.

Reply:
Sounds good, yeah those statisticGroupIDs will be handy for the next issue.